### PR TITLE
Fix <meta charset> with single quotes

### LINKED ISSUE #144: Quoting issue on charset detection
When trying to scrape ted.com:

```
2021-07-08 15:14:04.662214650 +03:00: [WARN] Charset 'utf-8' not supported for https://ted.com/attend/ted-on-screen, defaulting to UTF-8
2021-07-08 15:14:05.480213483 +03:00: [WARN] Charset 'utf-8' not supported for https://ted.com/participate/nominate, defaulting to UTF-8
2021-07-08 15:14:06.454700388 +03:00: [WARN] Charset 'utf-8' not supported for https://ted.com/participate/organize-a-local-tedx-event, defaulting to UTF-8
2021-07-08 15:14:07.116944837 +03:00: [WARN] Charset 'utf-8' not supported for https://ted.com/participate/translate, defaulting to UTF-8
```

Not a big deal, but kind of annoying. Also, I am not really sure how to fix it, since this method returns `'utf-8'`, not `utf-8`
https://github.com/Skallwar/suckit/blob/master/src/scraper.rs#L99

### PR DESCRIPTION
**Pull request by lhvy**
Resolves #144 with a small change to the regex to account for single or double quotes

> P.S. I'm participating in [Hacktoberfest 2021](https://hacktoberfest.digitalocean.com/). If this PR is up to standard and merged, I'd appreciate if the `hacktoberfest-accepted` label could be added. Thanks!

### CODE REVIEW

**Review by CohenArthur — COMMENTED** [review]
Could you add tests for this? Thanks!

**Comment by lhvy** [comment]
I'm unsure how to do this... The tests don't fail on the warn message, and suckit seems to scrape `<meta charset='utf-8'>` and output `<meta charset="utf-8">`.

**Comment by CohenArthur** [comment]
@lhvy Here's a test if you want to add it to the `scraper.rs` file:

```rust
    #[test]
    fn test_charset_parsing() {
        assert_eq!(Scraper::find_charset("<meta charset=\"UTF-8\">".as_bytes(), None), Some(String::from("utf-8")));
        assert_eq!(Scraper::find_charset("<meta charset='UTF-8'>".as_bytes(), None), Some(String::from("utf-8")));
    }
```
This just checks that the function returns the correct charset whether the value is between double quotes or single quotes

**Review by CohenArthur — APPROVED** [review]
Thanks!

File: src/scraper.rs [hunk 1]
```diff
@@ -438,4 +438,24 @@ mod tests {
...
+            Some("utf-8".to_string())
+        );
+        assert_eq!(
+            Scraper::find_charset(b"<meta charset=\"windows-1252\">", None),
+            Some("windows-1252".to_string())
+        );
+        assert_eq!(
+            Scraper::find_charset(b"<meta charset=\'windows-1252\'>", None),
+            Some("windows-1252".to_string())
+        );
```
Comment (Skallwar) [inline src/scraper.rs:459]: I would prefer to separate this into sub tests

**Review by Skallwar — CHANGES_REQUESTED** [review]
Great work, thanks !

File: src/scraper.rs (hunk 1 above)
Comment (lhvy) [reply src/scraper.rs]: Sure! I split them into separate tests for the double and single quotes.

**Review by Skallwar — APPROVED** [review]

**Pull request merged**